### ISSUE TITLE
Dose this project provide an interface for regression task?

### ISSUE THREAD

**Issue by bloom77** [mdeff/cnn_graph#9]
Hi,
I have read the paper  " Convolutional Neural Networks on Graphs with Fast Localized Spectral Filtering" and the usage notebook for figuring out the use of this models.
But I find that the tasks in your paper and the usage notebook are all aiming for classification.
Dose this project provide an interface for regression task?
Or do I have to write this interface by myself?

Thanks.

**Comment by mdeff** [comment]
Hi, thanks for your interest. The actual code was not used for regression, but it's just a matter of changing the classification loss function in `models.py` to whichever loss matters to you. Now it looks like:
```python
cross_entropy = tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=labels)
cross_entropy = tf.reduce_mean(cross_entropy)
loss = cross_entropy + regularization
```
You probably won't need any coarsening either, just set the pooling sizes `p` to 0.

**Comment by bloom77** [comment]
Thanks for answering.
I will have a try.
